README.md

```diff
@@ -13,17 +13,40 @@ Check out the `examples` folder for detailed descriptions and code examples for:
 
 # Installation
 
-Phoenics can be installed via pip
+## Environment (Optional)
+
+It is suggested to have a fresh environment in order to avoid to have pip modules compatibility issues. This is optional but recommended.
+
+This can be done with Venv or Anaconda.
+
+### Venv
+```bash
+python3.6 -m venv phoenics-env
+source phoenics-env/bin/activate
 ```
-  apt-get install python-pip
-  pip install phoenics
+A `venv` folder will be created in the current directory and the virtual enviroment will be activated.
+
+### Anaconda
+```bash
+conda create --name phoenics-env python=3.6
```

Comment:
Do we necessarily require python 3.6? I thought it would run just fine in 3.7 as well? Can we test if later version are supported too? That way, we could recommend 3.6+

Reply:
It works with Python 3.7.6

Reply:
Removed Python version from the creation of the environment. A note has been added for `edward` backend compatibility.